Source/Android/app/build.gradle

```diff
@@ -3,6 +3,14 @@ plugins {
     id 'org.jetbrains.kotlin.android'
 }
 
+task copyProfile (type: Copy) {
+    description('Copies a generated baseline profile text file from managed device to src/main in the app module.')
+    from('../benchmark/build/outputs/managed_device_android_test_additional_output/pixel6Api31')
```

Comment:
Probably `project(':benchmark').file('build/outputs/managed_device_android_test_additional_output/pixel6Api31'`) would be a better choice.

Is the `pixel6Api31` part generic, or specific to your device?

Reply:
It has to match the device declared in the build.gradle file in the benchmark module.

Reply:
Oh, I see, it's using an emulated device, not a physical one (e.g. via USB). Which... I guess is an obvious thing if it's being used on CI, but I still didn't realize it. That part makes sense now.